app/controllers/katello/concerns/organizations_controller_extensions.rb

```diff
@@ -3,8 +3,14 @@ module Concerns
     module OrganizationsControllerExtensions
       extend ActiveSupport::Concern
       include ForemanTasks::Triggers
+      include Foreman::Controller::Flash
 
       module Overrides
+        def edit
```

Comment:
Can you add a test for this method? Since we're overriding foreman behavior it's a critical place to test!